fast64_internal/f3d/f3d_gbi.py

```diff
@@ -3347,6 +3347,8 @@ def to_c(self, static=True):
             header += "segmented_to_virtual(" + str(self.matrix) + ")"
         else:
             header += str(self.matrix)
+            if bpy.context.scene.fast64.oot.shipOfHarkinianCompatible:
```

Comment:
I wouldn't add game specific code to f3d_gbi or f3d_writer if possible. For modifying material commands, you should look at `oot_model_classes.py -> OOTModel -> onMaterialCommandsBuilt`. `gfxList.commands` is a list of commands that you can iterate through, you can check for `isinstance(SPDisplayList)` and then modify that.